.github/workflows/release.yml

```diff
@@ -0,0 +1,100 @@
+name: lce-release
+
+# on:
+#   release:
+#     tags:
+#       - v*
+on: [push]
+
+jobs:
+  macos-release-wheel:
+    name: Build release wheels for macOS
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7']
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Bazelisk
+        run: |
+          curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/bazelisk-darwin-amd64 > /usr/local/bin/bazelisk
+          chmod +x /usr/local/bin/bazelisk
+      - name: Build macOS wheels
+        run: |
+          python --version
+          python -m pip install delocate wheel setuptools numpy six --no-cache-dir
+
+          ./configure.sh <<< $'yes\n'
+
+          bazelisk build :build_pip_pkg
+          bazel-bin/build_pip_pkg artifacts
+
+          for f in artifacts/*.whl; do
+            delocate-wheel -w wheelhouse $f
+          done
+        shell: bash
+      - uses: actions/upload-artifact@v1
+        with:
+          name: ${{ runner.os }}-wheels
+          path: wheelhouse
+
+  manylinux-release-wheel:
+    name: Build release wheels for manylinux2010
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7']
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Build manylinux2010 wheels
+        run: |
+          docker run -e PYTHON_VERSION=${{ matrix.python-version }} -v ${PWD}:/compute-engine -w /compute-engine \
+            tensorflow/tensorflow:custom-op-ubuntu16 \
+            .github/tools/release_linux.sh
+
+          sudo apt-get -y -qq install patchelf --no-install-recommends
+          python -m pip install auditwheel==2.0.0
+
+          for f in artifacts/*.whl; do
+            auditwheel show $f || true
+            # auditwheel repair fails likely due similar reasons as
+            # https://github.com/tensorflow/addons/pull/435
+            # auditwheel repair --plat manylinux2010_x86_64 $f
```

Comment:
Not sure about this, but wheels seem to work fine without it

Reply:
:+1: , but we should still have several people with different OS versions check the wheel.

Reply:
Agreed. I fairly am confident about the wheels though, since auditwheel only complains about our own shared library which we definitely need in the wheel.